### PR TITLE
[Keyboard]  Add hotdox76v2 support

### PR DESCRIPTION
**Pull request by drashna**
## Description

It keeps on coming up, from time to time, so here.

## Types of Changes

- [x] Keyboard (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hotdox76v2/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# hotdox76
+
+![hotdox76](imgur.com image replace me!)
```
Comment (keyboard-magpie) [inline keyboards/hotdox76v2/readme.md:3]: pleeeeeease replace me

**Review by keyboard-magpie — CHANGES_REQUESTED** [review]

File: keyboards/hotdox76v2/readme.md (hunk 1 above)
Comment (drashna) [reply keyboards/hotdox76v2/readme.md:3]: done: 
```suggestion
```

**Review by keyboard-magpie — APPROVED** [review]

File: keyboards/hotdox76v2/hotdox76v2.c [hunk 2]
```diff
@@ -0,0 +1,308 @@
+// Copyright 2021 JasonRen(biu)
```
Comment (spidey3) [inline keyboards/hotdox76v2/hotdox76v2.c:1]: Is it intentional to have different copyright holder here (and in the other files below)?

File: keyboards/hotdox76v2/readme.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# hotdox76
+
+
+*A short description of the keyboard/project*
```
Comment (zvecr) [inline keyboards/hotdox76v2/readme.md:4]: Remove?

File: keyboards/hotdox76v2/readme.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+* Hardware Supported: Hotdox 76v2
+Hardware Availability: https://github.com/Oh-My-Mechanical-Keyboard 
+
+Make example for this keyboard (after setting up your build environment):
+
+    make hotdox76:default
+
+Flashing example for this keyboard:
+
+    make hotdox76:default:flash
```
Comment (zvecr) [inline keyboards/hotdox76v2/readme.md:16]:
```suggestion
    make hotdox76v2:default

Flashing example for this keyboard:

    make hotdox76v2:default:flash
```

File: keyboards/hotdox76v2/readme.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# hotdox76
+
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Drashna Jael're](https://github.com/Drashna Jael're)
+* Hardware Supported: Hotdox 76v2
+Hardware Availability: https://github.com/Oh-My-Mechanical-Keyboard
```
Comment (zvecr) [inline keyboards/hotdox76v2/readme.md:8]:
```suggestion
* Hardware Availability: https://github.com/Oh-My-Mechanical-Keyboard 
```

File: keyboards/hotdox76v2/info.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (zvecr) [inline keyboards/hotdox76v2/info.json:1]: No layout macros?

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/hotdox76v2/info.json (hunk 6 above)
Comment (drashna) [reply keyboards/hotdox76v2/info.json:1]: source fork didn't have one, was being lazy.  Added, and verified that it's correct, codewise.

File: keyboards/hotdox76v2/hotdox76v2.h [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+    { KC_NO, R601,  R602,  R603,  R604,  R605,  R606  }, \
+    { R700,  R701,  R702,  R703,  R704,  KC_NO, KC_NO }, \
+    { R800,  R801,  R802,  R803,  R804,  R805,  R806  }, \
+    { R900,  R901,  R902,  R903,  R904,  R905,  KC_NO }, \
+    { R100,  R101,  R102,  R103,  R104,  R105,  R106  }, \
+    { R110,  R111,  R112,  R113,  R114,  R115,  R116  }  \
+}
+
+*/
+
```
Comment (zvecr) [inline keyboards/hotdox76v2/hotdox76v2.h:70]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

**Comment by razamatan** [comment]
❤️

**Pull request merged**